README.md

```diff
@@ -1,2 +1,437 @@
+    'KeyName': 'kp-us-east',
+    'NetworkInterfaces': [{'DeviceIndex': 0,
+      'Groups': ['sg-7c227019'],
+      'SubnetId': 'subnet-db7ea2c5',
+      'PrivateIpAddress': '80.141.44.57'}]},
+   'CreditSpecification': {'CpuCredits': 'standard'},
+   'CpuOptions': {'CoreCount': 2, 'ThreadsPerCore': 2},
+   'LaunchTemplateId': 'lt-068f72b72934aff71',
+   'VersionNumber': 2},
+  {'CreateTime': datetime.datetime(2017, 11, 20, 15, 0, 2),
+   'DefaultVersion': False,
+   'LaunchTemplateData': {'ImageId': 'ami-2661cf7b',
+    'KeyName': 'kp-us-east',
+    'NetworkInterfaces': [{'DeviceIndex': 0,
+      'Groups': ['sg-7c227019'],
+      'SubnetId': 'subnet-a4579fe6',
+      'Ipv6Addresses': [{'Ipv6Address': '2004:90aa:3c4c:c973:a318:af08:58a0:997c'},
+       {'Ipv6Address': 'b635:26ad:8fdf:a274:88dc:cf8c:47df:26b7'}],
+      'Ipv6AddressCount': 2,
+      'PrivateIpAddress': '80.141.152.44'}]},
+   'CpuOptions': {'CoreCount': 3, 'ThreadsPerCore': 1},
+   'CreditSpecification': {'CpuCredits': 'standard'},
+   'LaunchTemplateId': 'lt-068f72b72934aff71',
+   'VersionNumber': 3},
+  {'CreateTime': datetime.datetime(2017, 11, 20, 15, 45, 33),
+   'DefaultVersion': False,
+   'LaunchTemplateData': {'ImageId': 'ami-cc3e8abf',
+    'KeyName': 'kp-us-east',
+    'NetworkInterfaces': [{'DeviceIndex': 0,
+      'Groups': ['sg-7c227019'],
+      'SubnetId': 'subnet-a4579fe6',
+      'Ipv6Addresses': [{'Ipv6Address': '4f08:ea60:17f9:3e89:4d66:2e8c:259c:d1a9'},
+       {'Ipv6Address': 'b635:26ad:8fdf:a274:88dc:cf8c:47df:26b7'},
+       {'Ipv6Address': 'eb7a:5a31:f899:dd8c:e566:3307:a45e:dcf6'}],
+      'Ipv6AddressCount': 3,
+      'PrivateIpAddress': '80.141.152.14'}]},
+   'CpuOptions': {'CoreCount': 4, 'ThreadsPerCore': 1},
+   'CreditSpecification': {'CpuCredits': 'unlimited'},
+   'LaunchTemplateId': 'lt-aaa68831cce2a8d91',
+   'VersionNumber': 4},
+  {'CreateTime': datetime.datetime(2017, 11, 20, 19, 4, 54),
+   'DefaultVersion': False,
+   'LaunchTemplateData': {'ImageId': 'ami-2f7ac02a',
+    'KeyName': 'kp-us-east',
+    'NetworkInterfaces': [{'DeviceIndex': 0,
+      'Groups': ['sg-1c628b25'],
+      'SubnetId': 'subnet-a4579fe6',
+      'Ipv6Addresses': [{'Ipv6Address': 'f486:915c:2be9:b0da:7d60:3fae:d65a:e8d8'},
+       {'Ipv6Address': 'eb7a:5a31:f899:dd8c:e566:3307:a45e:dcf6'}],
+      'Ipv6AddressCount': 2,
+      'PrivateIpAddress': '80.141.152.136'}]},
+   'CpuOptions': {'CoreCount': 3, 'ThreadsPerCore': 2},
+   'CreditSpecification': {'CpuCredits': 'standard'},
+   'LaunchTemplateId': 'lt-aaa68831cce2a8d91',
+   'VersionNumber': 5}],
+ 'ResponseMetadata': {'RequestId': '856F4F9E52336fA4',
+  'HostId': '118e71806df8f025567be1b09c071e900',
+  'HTTPStatusCode': 200,
+  'RetryAttempts': 0}}
+```
+
+If we want to get data that have `AssociatePublicIpAddress`
+set to `True`, we can do the following:
+
+```python
+>>> from query_filter import q_filter, q_item
+>>> results = q_filter(
+        data["LaunchTemplateVersions"],
```

Comment:
It seems that this should refer to the dictionary `versions_data` defined above, rather than `data`.